.github/workflows/sync-ceremony-to-main.yml

```diff
@@ -13,12 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This workflow is triggered when a ceremony branch is completed, creating a PR
-# that merges the completed ceremony branch to main.
-# As a result, this updates the files in `repository/repository/**` at main,
-# thereby automatically triggering the `sync-main-to-preprod.yml` workflow.
+# This workflow is triggered when a ceremony branch is completed, and will:
+# - Create a PR that merges the completed ceremony branch to main
+# - Sync the main branch to the GCS preprod bucket
```

Comment:
Technically it's not `main` that's synced to preprod bucket, right? It's the ceremony branch that's about to be merged to main. So something like `Sync the ceremony branch to the GCS preprod bucket` would be better I think.

Reply:
Updated